alws/dramatiq/products.py

```diff
@@ -188,8 +191,12 @@ async def set_platform_for_build_repos(
     # name of a build's repo:
     # some-platform-x86_64-some-build-id-debug-br
     repos_per_platform = {
-        f"{task.platform.name}-{repo.arch}-{build.id}-"
-        f"{repo_debug_dict[repo.debug]}": task.platform
+        # fmt: off
+        (
+            f"{task.platform.name}-{repo.arch}-{build.id}"
+            f"-{repo_debug_dict[repo.debug]}"
+        ): task.platform
+        # fmt: on
```

Comment:
I forgot to leave a comment there earlier regarding removing `fmt: off`
could you remove it there too?

Reply:
Done